apps/mobile/src/components/Alert/Alert.tsx

```diff
@@ -83,6 +84,7 @@ export const Alert = ({
   testID,
   info,
   orientation = 'center',
+  bgcolor = '$background',
```

Comment:
This is a bit weird. The type of the alert represent the Tamagui theme for that element. The tamagui theme has a background and color. Now the component signature changes - you say you want a warning and the warning has a predefined background, but you can override it by passing bgcolor. 

This feels weird. Why can't we have another theme?

Reply:
I also think its not ideal. We use the Alert type for both the theme and the icons though so it doesn't have the theme type. I will remove it again. We lose the background in light mode but I think its not too bad.